src/app/tests/suites/TestCommissionerNodeId.yaml

```diff
@@ -0,0 +1,235 @@
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Read the fabric ID from the alpha fabric"
+      identity: "alpha"
+      cluster: "Operational Credentials"
+      command: "readAttribute"
+      attribute: "CurrentFabricIndex"
+      response:
+          saveAs: alphaIndex
+
+    - label: "Read the fabric ID from the beta fabric"
+      identity: "beta"
+      cluster: "Operational Credentials"
+      command: "readAttribute"
+      attribute: "CurrentFabricIndex"
+      response:
+          saveAs: betaIndex
+
+    - label: "Read the fabric ID from the gamma fabric"
+      identity: "gamma"
+      cluster: "Operational Credentials"
+      command: "readAttribute"
+      attribute: "CurrentFabricIndex"
+      response:
+          saveAs: gammaIndex
+
+    - label: "Read the ACL from alpha and check commissioner node id"
+      identity: "alpha"
+      command: "readAttribute"
+      attribute: "ACL"
+      response:
+          value: [
+                  {
+                      FabricIndex: alphaIndex,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [commissionerNodeId],
+                      Targets: null,
+                  },
+              ]
+
+    - label: "Read the ACL from beta and check commissioner node id"
+      identity: "beta"
+      command: "readAttribute"
+      attribute: "ACL"
+      response:
+          value: [
+                  {
+                      FabricIndex: betaIndex,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [commissionerNodeId],
+                      Targets: null,
+                  },
+              ]
+
+    - label: "Read the ACL from gamma and check commissioner node id"
+      identity: "gamma"
+      command: "readAttribute"
+      attribute: "ACL"
+      response:
+          value: [
+                  {
+                      FabricIndex: gammaIndex,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [commissionerNodeId],
+                      Targets: null,
+                  },
+              ]
+
+    - label: "Write the ACL using the commissioner node id value"
+      identity: "beta"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  {
+                      FabricIndex: betaIndex,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [commissionerNodeId],
+                      Targets: null,
+                  },
+              ]
+
+    - label: "Write the ACL using the commissioner node id value"
+      identity: "gamma"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  {
+                      FabricIndex: betaIndex,
```

Comment:
The betaIndex here is confusing.  If we can just leave FabricIndex out of the writes, great.  If not (i.e. codegen barfs), we should either use 0 (preferred) or gammaIndex here.

Reply:
whoops yeah. This was a typo. It's really just to appease the yaml parser - it's not checked. Will change.